.eslintrc.json

```diff
@@ -35,6 +35,7 @@
     }],
     "func-names": ["warn", "as-needed"],
     "jsx-a11y/label-has-for": 0,
-    "import/prefer-default-export": 0
+    "import/prefer-default-export": 0,
+    "no-return-assign": 0
```

Comment:
I think we can get rid of this since it is a common patter to return assignments for react refs
```
<input ref={(input) => this.input = input} />
```